cms/djangoapps/contentstore/utils.py

```diff
@@ -160,7 +160,10 @@ def get_lms_link_for_certificate_web_view(user_id, course_key, mode):
 
 def course_image_url(course):
     """Returns the image url for the course."""
-    loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    try:
+        loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+    except InvalidKeyError:
+        return None
```

Comment:
@cpennington Right, I see what you mean. 
So what is the better way to to handle this, What should we return? because you are right, because it would be used in the templates.